.github/dependabot.yml

```diff
@@ -0,0 +1,12 @@
+version: 2
+updates:
+  - package-ecosystem: "hex"
```

Comment:
In addition to elixir dependencies, we have [npm packages](https://github.com/artsy/aprd/tree/fed6a01199e871dad4e09aad65bc7135c06a3814/assets) for the dashboard UI. And looks like [current security alerts](https://github.com/artsy/aprd/security/dependabot) are all about npm packages. Perhaps we want to include npm as well?

Reply:
@starsirius 🙏 thanks for pointing that out. totally missed it.